docs/streams/upgrade-guide.html

```diff
@@ -93,6 +93,9 @@ <h1>Upgrade Guide and API Changes</h1>
     </p>
 
     <h3><a id="streams_api_changes_300" href="#streams_api_changes_300">Streams API changes in 3.0.0</a></h3>
+    <p>
+        We applyed UnknownStateStoreException to <code>KafkaStreams#store()</code>: If the specified store name does not exist in the topology, UnknownStateStoreException will be thrown instead InvalidStateStoreException(<a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-216%3A+IQ+should+throw+different+exceptions+for+different+errors">KIP-216</a>).
```

Comment:
```suggestion
        A new exception may be thrown from <code>KafkaStreams#store()</code>. If the specified store name does not exist in the topology, an UnknownStateStoreException will be thrown instead of the former InvalidStateStoreException. See <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-216%3A+IQ+should+throw+different+exceptions+for+different+errors">KIP-216</a> for more information.
```

Reply:
Thanks @ableegoldman !